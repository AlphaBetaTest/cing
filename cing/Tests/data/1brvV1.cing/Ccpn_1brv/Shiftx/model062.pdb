REMARK PDB file of molecule Ccpn_1brv                                 
ATOM      1  N   VAL A 171       5.057  -0.060   4.915  1.00  0.00              
ATOM      2  CA  VAL A 171       4.415   0.549   3.724  1.00  0.00              
ATOM      3  HA  VAL A 171       3.961   1.525   3.982  1.00  0.00              
ATOM      4  CB  VAL A 171       5.586   0.895   2.734  1.00  0.00              
ATOM      5  HB  VAL A 171       5.083   1.313   1.860  1.00  0.00              
ATOM      6  CG1 VAL A 171       6.548   1.999   3.239  1.00  0.00              
ATOM      7 HG11 VAL A 171       6.013   2.941   3.464  1.00  0.00              
ATOM      8 HG12 VAL A 171       7.084   1.698   4.158  1.00  0.00              
ATOM      9 HG13 VAL A 171       7.326   2.244   2.492  1.00  0.00              
ATOM     10  CG2 VAL A 171       6.433  -0.279   2.187  1.00  0.00              
ATOM     11 HG21 VAL A 171       5.800  -1.026   1.677  1.00  0.00              
ATOM     12 HG22 VAL A 171       7.164   0.060   1.428  1.00  0.00              
ATOM     13 HG23 VAL A 171       7.000  -0.803   2.978  1.00  0.00              
ATOM     14  C   VAL A 171       3.194  -0.275   3.147  1.00  0.00              
ATOM     15  O   VAL A 171       3.264  -0.704   1.995  1.00  0.00              
ATOM     16  N   PRO A 172       2.033  -0.504   3.842  1.00  0.00              
ATOM     17  CD  PRO A 172       1.938  -0.433   5.318  1.00  0.00              
ATOM     18  CA  PRO A 172       0.854  -1.178   3.225  1.00  0.00              
ATOM     19  HA  PRO A 172       1.163  -1.873   2.416  1.00  0.00              
ATOM     20  CB  PRO A 172       0.344  -2.030   4.402  1.00  0.00              
ATOM     21  HB2 PRO A 172      -0.731  -2.285   4.323  1.00  0.00              
ATOM     22  HB3 PRO A 172       0.890  -2.994   4.424  1.00  0.00              
ATOM     23  CG  PRO A 172       0.672  -1.223   5.661  1.00  0.00              
ATOM     24  HG2 PRO A 172      -0.155  -0.534   5.893  1.00  0.00              
ATOM     25  HG3 PRO A 172       0.803  -1.865   6.553  1.00  0.00              
ATOM     26  HD2 PRO A 172       1.887   0.612   5.682  1.00  0.00              
ATOM     27  HD3 PRO A 172       2.801  -0.918   5.811  1.00  0.00              
ATOM     28  C   PRO A 172      -0.193  -0.151   2.663  1.00  0.00              
ATOM     29  O   PRO A 172      -0.698   0.715   3.392  1.00  0.00              
ATOM     30  N   CYS A 173      -0.560  -0.276   1.374  1.00  0.00              
ATOM     31  H   CYS A 173      -0.140  -1.054   0.856  1.00  0.00              
ATOM     32  CA  CYS A 173      -1.586   0.583   0.708  1.00  0.00              
ATOM     33  HA  CYS A 173      -1.251   1.622   0.727  1.00  0.00              
ATOM     34  CB  CYS A 173      -1.561   0.247  -0.783  1.00  0.00              
ATOM     35  HB2 CYS A 173      -0.983  -0.650  -1.015  1.00  0.00              
ATOM     36  HB3 CYS A 173      -2.574   0.093  -1.130  1.00  0.00              
ATOM     37  SG  CYS A 173      -0.750   1.529  -1.733  1.00  0.00              
ATOM     38  C   CYS A 173      -3.011   0.635   1.359  1.00  0.00              
ATOM     39  O   CYS A 173      -3.696   1.661   1.266  1.00  0.00              
ATOM     40  N   SER A 174      -3.442  -0.442   2.040  1.00  0.00              
ATOM     41  H   SER A 174      -2.791  -1.235   1.985  1.00  0.00              
ATOM     42  CA  SER A 174      -4.643  -0.422   2.933  1.00  0.00              
ATOM     43  HA  SER A 174      -5.526  -0.277   2.311  1.00  0.00              
ATOM     44  CB  SER A 174      -4.765  -1.810   3.608  1.00  0.00              
ATOM     45  HB2 SER A 174      -5.731  -1.881   4.146  1.00  0.00              
ATOM     46  HB3 SER A 174      -4.804  -2.609   2.846  1.00  0.00              
ATOM     47  OG  SER A 174      -3.692  -2.076   4.518  1.00  0.00              
ATOM     48  HG  SER A 174      -3.808  -1.488   5.272  1.00  0.00              
ATOM     49  C   SER A 174      -4.717   0.723   4.002  1.00  0.00              
ATOM     50  O   SER A 174      -5.795   1.222   4.324  1.00  0.00              
ATOM     51  N   THR A 175      -3.551   1.117   4.524  1.00  0.00              
ATOM     52  H   THR A 175      -2.746   0.652   4.084  1.00  0.00              
ATOM     53  CA  THR A 175      -3.394   2.202   5.538  1.00  0.00              
ATOM     54  HA  THR A 175      -4.359   2.330   6.068  1.00  0.00              
ATOM     55  CB  THR A 175      -2.364   1.724   6.607  1.00  0.00              
ATOM     56  HB  THR A 175      -2.395   2.447   7.448  1.00  0.00              
ATOM     57  OG1 THR A 175      -1.036   1.722   6.087  1.00  0.00              
ATOM     58  HG1 THR A 175      -1.069   1.339   5.200  1.00  0.00              
ATOM     59  CG2 THR A 175      -2.655   0.322   7.185  1.00  0.00              
ATOM     60 HG21 THR A 175      -3.705   0.241   7.526  1.00  0.00              
ATOM     61 HG22 THR A 175      -2.519  -0.465   6.418  1.00  0.00              
ATOM     62 HG23 THR A 175      -1.991   0.073   8.026  1.00  0.00              
ATOM     63  C   THR A 175      -3.058   3.630   4.967  1.00  0.00              
ATOM     64  O   THR A 175      -3.101   4.612   5.714  1.00  0.00              
ATOM     65  N   CYS A 176      -2.712   3.763   3.671  1.00  0.00              
ATOM     66  H   CYS A 176      -2.810   2.926   3.091  1.00  0.00              
ATOM     67  CA  CYS A 176      -2.539   5.057   2.980  1.00  0.00              
ATOM     68  HA  CYS A 176      -2.079   5.799   3.663  1.00  0.00              
ATOM     69  CB  CYS A 176      -1.529   4.713   1.871  1.00  0.00              
ATOM     70  HB2 CYS A 176      -1.889   4.008   1.130  1.00  0.00              
ATOM     71  HB3 CYS A 176      -1.310   5.574   1.227  1.00  0.00              
ATOM     72  SG  CYS A 176       0.015   4.116   2.589  1.00  0.00              
ATOM     73  C   CYS A 176      -3.904   5.666   2.493  1.00  0.00              
ATOM     74  O   CYS A 176      -4.098   6.877   2.594  1.00  0.00              
ATOM     75  N   GLU A 177      -4.822   4.829   1.959  1.00  0.00              
ATOM     76  H   GLU A 177      -4.415   3.946   1.646  1.00  0.00              
ATOM     77  CA  GLU A 177      -6.230   5.175   1.604  1.00  0.00              
ATOM     78  HA  GLU A 177      -6.492   4.414   0.850  1.00  0.00              
ATOM     79  CB  GLU A 177      -7.163   4.885   2.813  1.00  0.00              
ATOM     80  HB2 GLU A 177      -8.189   4.930   2.408  1.00  0.00              
ATOM     81  HB3 GLU A 177      -7.024   3.831   3.118  1.00  0.00              
ATOM     82  CG  GLU A 177      -7.133   5.807   4.067  1.00  0.00              
ATOM     83  HG2 GLU A 177      -6.940   6.858   3.779  1.00  0.00              
ATOM     84  HG3 GLU A 177      -8.133   5.820   4.534  1.00  0.00              
ATOM     85  CD  GLU A 177      -6.159   5.380   5.161  1.00  0.00              
ATOM     86  OE1 GLU A 177      -6.238   4.318   5.770  1.00  0.00              
ATOM     87  OE2 GLU A 177      -5.180   6.298   5.375  1.00  0.00              
ATOM     88  C   GLU A 177      -6.544   6.510   0.869  1.00  0.00              
ATOM     89  O   GLU A 177      -7.466   7.262   1.196  1.00  0.00              
ATOM     90  N   GLY A 178      -5.784   6.730  -0.199  1.00  0.00              
ATOM     91  H   GLY A 178      -4.887   6.242  -0.116  1.00  0.00              
ATOM     92  CA  GLY A 178      -5.867   7.979  -0.978  1.00  0.00              
ATOM     93  HA2 GLY A 178      -5.846   7.675  -2.034  1.00  0.00              
ATOM     94  HA3 GLY A 178      -6.833   8.471  -0.829  1.00  0.00              
ATOM     95  C   GLY A 178      -4.748   9.022  -0.771  1.00  0.00              
ATOM     96  O   GLY A 178      -4.742  10.035  -1.472  1.00  0.00              
ATOM     97  N   ASN A 179      -3.786   8.789   0.145  1.00  0.00              
ATOM     98  H   ASN A 179      -3.760   7.813   0.474  1.00  0.00              
ATOM     99  CA  ASN A 179      -2.609   9.664   0.303  1.00  0.00              
ATOM    100  HA  ASN A 179      -2.854  10.704   0.009  1.00  0.00              
ATOM    101  CB  ASN A 179      -2.006   9.523   1.714  1.00  0.00              
ATOM    102  HB2 ASN A 179      -2.775   9.338   2.489  1.00  0.00              
ATOM    103  HB3 ASN A 179      -1.364   8.619   1.663  1.00  0.00              
ATOM    104  CG  ASN A 179      -1.100  10.680   2.103  1.00  0.00              
ATOM    105  OD1 ASN A 179       0.066  10.740   1.734  1.00  0.00              
ATOM    106  ND2 ASN A 179      -1.613  11.641   2.817  1.00  0.00              
ATOM    107 HD21 ASN A 179      -2.593  11.531   3.091  1.00  0.00              
ATOM    108 HD22 ASN A 179      -0.957  12.391   3.042  1.00  0.00              
ATOM    109  C   ASN A 179      -1.590   9.162  -0.706  1.00  0.00              
ATOM    110  O   ASN A 179      -0.937   8.127  -0.533  1.00  0.00              
ATOM    111  N   LEU A 180      -1.446   9.923  -1.767  1.00  0.00              
ATOM    112  H   LEU A 180      -2.069  10.731  -1.838  1.00  0.00              
ATOM    113  CA  LEU A 180      -0.637   9.468  -2.890  1.00  0.00              
ATOM    114  HA  LEU A 180      -1.055   8.456  -2.996  1.00  0.00              
ATOM    115  CB  LEU A 180      -1.022  10.431  -4.024  1.00  0.00              
ATOM    116  HB2 LEU A 180      -1.202  11.450  -3.606  1.00  0.00              
ATOM    117  HB3 LEU A 180      -0.161  10.506  -4.657  1.00  0.00              
ATOM    118  CG  LEU A 180      -2.187  10.011  -4.925  1.00  0.00              
ATOM    119  HG  LEU A 180      -2.994   9.818  -4.220  1.00  0.00              
ATOM    120  CD1 LEU A 180      -2.604  11.151  -5.866  1.00  0.00              
ATOM    121 HD11 LEU A 180      -2.874  12.069  -5.310  1.00  0.00              
ATOM    122 HD12 LEU A 180      -1.799  11.423  -6.575  1.00  0.00              
ATOM    123 HD13 LEU A 180      -3.490  10.879  -6.470  1.00  0.00              
ATOM    124  CD2 LEU A 180      -1.903   8.743  -5.753  1.00  0.00              
ATOM    125 HD21 LEU A 180      -1.658   7.871  -5.120  1.00  0.00              
ATOM    126 HD22 LEU A 180      -2.776   8.449  -6.365  1.00  0.00              
ATOM    127 HD23 LEU A 180      -1.052   8.884  -6.447  1.00  0.00              
ATOM    128  C   LEU A 180       0.888   9.234  -2.677  1.00  0.00              
ATOM    129  O   LEU A 180       1.492   8.418  -3.364  1.00  0.00              
ATOM    130  N   ALA A 181       1.456   9.916  -1.688  1.00  0.00              
ATOM    131  H   ALA A 181       0.725  10.435  -1.201  1.00  0.00              
ATOM    132  CA  ALA A 181       2.808   9.630  -1.170  1.00  0.00              
ATOM    133  HA  ALA A 181       3.479   9.536  -2.036  1.00  0.00              
ATOM    134  CB  ALA A 181       3.252  10.844  -0.362  1.00  0.00              
ATOM    135  HB1 ALA A 181       3.251  11.755  -0.984  1.00  0.00              
ATOM    136  HB2 ALA A 181       2.592  11.004   0.512  1.00  0.00              
ATOM    137  HB3 ALA A 181       4.279  10.686   0.012  1.00  0.00              
ATOM    138  C   ALA A 181       2.873   8.299  -0.344  1.00  0.00              
ATOM    139  O   ALA A 181       3.716   7.457  -0.638  1.00  0.00              
ATOM    140  N   CYS A 182       1.981   8.071   0.642  1.00  0.00              
ATOM    141  H   CYS A 182       1.170   8.699   0.592  1.00  0.00              
ATOM    142  CA  CYS A 182       1.858   6.759   1.355  1.00  0.00              
ATOM    143  HA  CYS A 182       2.743   6.512   1.905  1.00  0.00              
ATOM    144  CB  CYS A 182       0.816   7.004   2.440  1.00  0.00              
ATOM    145  HB2 CYS A 182       0.976   7.963   2.941  1.00  0.00              
ATOM    146  HB3 CYS A 182      -0.171   7.087   1.990  1.00  0.00              
ATOM    147  SG  CYS A 182       0.745   5.651   3.621  1.00  0.00              
ATOM    148  C   CYS A 182       1.634   5.528   0.423  1.00  0.00              
ATOM    149  O   CYS A 182       2.309   4.499   0.507  1.00  0.00              
ATOM    150  N   LEU A 183       0.692   5.708  -0.493  1.00  0.00              
ATOM    151  H   LEU A 183       0.187   6.601  -0.403  1.00  0.00              
ATOM    152  CA  LEU A 183       0.420   4.803  -1.618  1.00  0.00              
ATOM    153  HA  LEU A 183       0.132   3.815  -1.221  1.00  0.00              
ATOM    154  CB  LEU A 183      -0.775   5.516  -2.285  1.00  0.00              
ATOM    155  HB2 LEU A 183      -0.486   6.550  -2.527  1.00  0.00              
ATOM    156  HB3 LEU A 183      -0.915   5.095  -3.238  1.00  0.00              
ATOM    157  CG  LEU A 183      -2.175   5.473  -1.667  1.00  0.00              
ATOM    158  HG  LEU A 183      -2.055   5.960  -0.686  1.00  0.00              
ATOM    159  CD1 LEU A 183      -3.139   6.247  -2.583  1.00  0.00              
ATOM    160 HD11 LEU A 183      -3.070   5.912  -3.637  1.00  0.00              
ATOM    161 HD12 LEU A 183      -4.191   6.110  -2.294  1.00  0.00              
ATOM    162 HD13 LEU A 183      -2.935   7.332  -2.580  1.00  0.00              
ATOM    163  CD2 LEU A 183      -2.724   4.047  -1.525  1.00  0.00              
ATOM    164 HD21 LEU A 183      -2.794   3.526  -2.500  1.00  0.00              
ATOM    165 HD22 LEU A 183      -2.090   3.419  -0.885  1.00  0.00              
ATOM    166 HD23 LEU A 183      -3.732   4.027  -1.073  1.00  0.00              
ATOM    167  C   LEU A 183       1.605   4.593  -2.634  1.00  0.00              
ATOM    168  O   LEU A 183       1.744   3.503  -3.185  1.00  0.00              
ATOM    169  N   SER A 184       2.431   5.627  -2.892  1.00  0.00              
ATOM    170  H   SER A 184       2.132   6.475  -2.391  1.00  0.00              
ATOM    171  CA  SER A 184       3.685   5.500  -3.710  1.00  0.00              
ATOM    172  HA  SER A 184       3.402   5.066  -4.689  1.00  0.00              
ATOM    173  CB  SER A 184       4.296   6.899  -3.991  1.00  0.00              
ATOM    174  HB2 SER A 184       5.036   6.802  -4.804  1.00  0.00              
ATOM    175  HB3 SER A 184       3.530   7.597  -4.360  1.00  0.00              
ATOM    176  OG  SER A 184       4.976   7.488  -2.880  1.00  0.00              
ATOM    177  HG  SER A 184       4.454   7.318  -2.079  1.00  0.00              
ATOM    178  C   SER A 184       4.782   4.548  -3.126  1.00  0.00              
ATOM    179  O   SER A 184       5.475   3.851  -3.869  1.00  0.00              
ATOM    180  N   LEU A 185       4.941   4.557  -1.795  1.00  0.00              
ATOM    181  H   LEU A 185       4.246   5.141  -1.305  1.00  0.00              
ATOM    182  CA  LEU A 185       5.860   3.635  -1.065  1.00  0.00              
ATOM    183  HA  LEU A 185       6.809   3.532  -1.599  1.00  0.00              
ATOM    184  CB  LEU A 185       6.114   4.228   0.355  1.00  0.00              
ATOM    185  HB2 LEU A 185       5.220   4.014   0.974  1.00  0.00              
ATOM    186  HB3 LEU A 185       6.929   3.648   0.823  1.00  0.00              
ATOM    187  CG  LEU A 185       6.438   5.738   0.497  1.00  0.00              
ATOM    188  HG  LEU A 185       5.610   6.294   0.004  1.00  0.00              
ATOM    189  CD1 LEU A 185       6.478   6.146   1.977  1.00  0.00              
ATOM    190 HD11 LEU A 185       5.521   5.926   2.486  1.00  0.00              
ATOM    191 HD12 LEU A 185       7.275   5.618   2.534  1.00  0.00              
ATOM    192 HD13 LEU A 185       6.657   7.231   2.096  1.00  0.00              
ATOM    193  CD2 LEU A 185       7.763   6.120  -0.182  1.00  0.00              
ATOM    194 HD21 LEU A 185       8.626   5.575   0.244  1.00  0.00              
ATOM    195 HD22 LEU A 185       7.739   5.898  -1.266  1.00  0.00              
ATOM    196 HD23 LEU A 185       7.974   7.202  -0.091  1.00  0.00              
ATOM    197  C   LEU A 185       5.408   2.150  -0.978  1.00  0.00              
ATOM    198  O   LEU A 185       6.242   1.249  -0.854  1.00  0.00              
ATOM    199  N   CYS A 186       4.094   1.913  -1.015  1.00  0.00              
ATOM    200  H   CYS A 186       3.555   2.704  -0.666  1.00  0.00              
ATOM    201  CA  CYS A 186       3.515   0.600  -0.840  1.00  0.00              
ATOM    202  HA  CYS A 186       4.048   0.248   0.048  1.00  0.00              
ATOM    203  CB  CYS A 186       2.026   0.938  -0.565  1.00  0.00              
ATOM    204  HB2 CYS A 186       1.636   0.151   0.081  1.00  0.00              
ATOM    205  HB3 CYS A 186       1.791   1.863  -0.018  1.00  0.00              
ATOM    206  SG  CYS A 186       1.095   0.923  -2.112  1.00  0.00              
ATOM    207  C   CYS A 186       3.643  -0.510  -1.933  1.00  0.00              
ATOM    208  O   CYS A 186       4.127  -0.337  -3.053  1.00  0.00              
ATOM    209  N   HIS A 187       3.083  -1.663  -1.549  1.00  0.00              
ATOM    210  H   HIS A 187       3.081  -1.734  -0.525  1.00  0.00              
ATOM    211  CA  HIS A 187       2.950  -2.859  -2.416  1.00  0.00              
ATOM    212  HA  HIS A 187       3.630  -2.760  -3.286  1.00  0.00              
ATOM    213  CB  HIS A 187       3.368  -4.179  -1.672  1.00  0.00              
ATOM    214  HB2 HIS A 187       2.505  -4.814  -1.464  1.00  0.00              
ATOM    215  HB3 HIS A 187       3.952  -4.764  -2.394  1.00  0.00              
ATOM    216  CG  HIS A 187       4.157  -4.129  -0.370  1.00  0.00              
ATOM    217  ND1 HIS A 187       3.567  -3.832   0.847  1.00  0.00              
ATOM    218  CD2 HIS A 187       5.543  -4.138  -0.272  1.00  0.00              
ATOM    219  CE1 HIS A 187       4.699  -3.642   1.590  1.00  0.00              
ATOM    220  NE2 HIS A 187       5.933  -3.841   1.022  1.00  0.00              
ATOM    221  HD2 HIS A 187       6.172  -4.255  -1.138  1.00  0.00              
ATOM    222  HE1 HIS A 187       4.601  -3.164   2.554  1.00  0.00              
ATOM    223  C   HIS A 187       1.489  -2.886  -3.012  1.00  0.00              
ATOM    224  O   HIS A 187       1.295  -2.495  -4.165  1.00  0.00              
ATOM    225  N   ILE A 188       0.456  -3.339  -2.260  1.00  0.00              
ATOM    226  H   ILE A 188       0.697  -4.030  -1.559  1.00  0.00              
ATOM    227  CA  ILE A 188      -0.937  -3.472  -2.749  1.00  0.00              
ATOM    228  HA  ILE A 188      -1.094  -2.707  -3.538  1.00  0.00              
ATOM    229  CB  ILE A 188      -1.015  -4.928  -3.360  1.00  0.00              
ATOM    230  HB  ILE A 188      -0.146  -5.057  -4.039  1.00  0.00              
ATOM    231  CG2 ILE A 188      -0.968  -6.086  -2.319  1.00  0.00              
ATOM    232 HG21 ILE A 188      -0.060  -6.044  -1.691  1.00  0.00              
ATOM    233 HG22 ILE A 188      -1.835  -6.064  -1.631  1.00  0.00              
ATOM    234 HG23 ILE A 188      -0.962  -7.080  -2.802  1.00  0.00              
ATOM    235  CG1 ILE A 188      -2.258  -5.058  -4.251  1.00  0.00              
ATOM    236 HG12 ILE A 188      -3.133  -5.100  -3.579  1.00  0.00              
ATOM    237 HG13 ILE A 188      -2.333  -4.115  -4.821  1.00  0.00              
ATOM    238  CD1 ILE A 188      -2.270  -6.232  -5.247  1.00  0.00              
ATOM    239 HD11 ILE A 188      -1.390  -6.209  -5.918  1.00  0.00              
ATOM    240 HD12 ILE A 188      -2.270  -7.213  -4.737  1.00  0.00              
ATOM    241 HD13 ILE A 188      -3.168  -6.205  -5.891  1.00  0.00              
ATOM    242  C   ILE A 188      -2.026  -3.146  -1.654  1.00  0.00              
ATOM    243  O   ILE A 188      -1.777  -3.205  -0.445  1.00  0.00              
ATOM    244  N   GLU A 189      -3.261  -2.821  -2.088  1.00  0.00              
ATOM    245  H   GLU A 189      -3.353  -2.712  -3.096  1.00  0.00              
ATOM    246  CA  GLU A 189      -4.422  -2.598  -1.183  1.00  0.00              
ATOM    247  HA  GLU A 189      -4.116  -1.866  -0.419  1.00  0.00              
ATOM    248  CB  GLU A 189      -5.612  -1.996  -1.986  1.00  0.00              
ATOM    249  HB2 GLU A 189      -5.875  -2.681  -2.817  1.00  0.00              
ATOM    250  HB3 GLU A 189      -6.516  -1.998  -1.343  1.00  0.00              
ATOM    251  CG  GLU A 189      -5.463  -0.555  -2.550  1.00  0.00              
ATOM    252  HG2 GLU A 189      -4.624  -0.522  -3.270  1.00  0.00              
ATOM    253  HG3 GLU A 189      -6.356  -0.309  -3.152  1.00  0.00              
ATOM    254  CD  GLU A 189      -5.269   0.607  -1.570  1.00  0.00              
ATOM    255  OE1 GLU A 189      -5.788   0.376  -0.334  1.00  0.00              
ATOM    256  OE2 GLU A 189      -4.687   1.640  -1.872  1.00  0.00              
ATOM    257  C   GLU A 189      -4.893  -3.877  -0.418  1.00  0.00              
TER     258      GLU A 189 
END   
