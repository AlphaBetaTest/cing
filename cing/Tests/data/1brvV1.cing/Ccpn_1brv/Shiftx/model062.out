# Entries marked with a * may have inaccurate shift predictions.
# Entries marked with a value -666 should be igonored
# Output format adjusted by GV 11 September 2007
  171   V  N      114.0584
  171   V  CA      60.5994
  171   V  CB      32.9119
  171   V  C      174.2607
  171   V  H        7.8976
  171   V  HA       4.0516
  171   V  HB       2.0262
  171   V  HG11     0.9233
  171   V  HG12     0.9513
  171   V  HG13     0.2335
  171   V  HG21     0.7181
  171   V  HG22     0.9520
  171   V  HG23     0.9289
  172   P  CA      60.9339
  172   P  CB      33.1194
  172   P  C      176.9111
  172   P  HA       4.8598
  172   P  HB2      2.3099
  172   P  HB3      2.2128
  172   P  HD2      3.7151
  172   P  HD3      3.7903
  172   P  HG2      2.0214
  172   P  HG3      2.0644
  173   C  N      121.2295
  173   C  CA      59.9169
  173   C  CB      40.7523
  173   C  C      177.1631
  173   C  H        8.8020
  173   C  HA       4.2779
  173   C  HB2      3.0436
  173   C  HB3      3.3289
  174   S  N      115.0775
  174   S  CA      61.4835
  174   S  CB      62.2914
  174   S  C      175.8423
  174   S  H        8.2281
  174   S  HA       4.0728
  174   S  HB2      4.0190
  174   S  HB3      4.0322
  175   T  N      110.4339
  175   T  CA      62.6081
  175   T  CB      69.4458
  175   T  C      175.8165
  175   T  H        7.6646
  175   T  HA       4.4211
  175   T  HB       4.2427
  175   T  HG21     1.2140
  175   T  HG22     1.2166
  175   T  HG23     1.2151
  176   C  N      120.0945
  176   C  CA      58.0575
  176   C  CB      41.0705
  176   C  C      173.5099
  176   C  H        8.1803
  176   C  HA       4.3638
  176   C  HB2      3.1151
  176   C  HB3      3.2054
  177   E  N      116.7423
  177   E  CA      57.4073
  177   E  CB      28.4619
  177   E  C      175.4879
  177   E  H        8.9500
  177   E  HA       3.4855
  177   E  HB2      2.1355
  177   E  HB3      2.0297
  177   E  HG2      2.2268
  177   E  HG3      2.3776
  178   G  N      104.3534
  178   G  CA      46.1359
  178   G  C      173.5701
  178   G  H        8.3055
  178   G  HA    -666.0000
  178   G  HA2      3.9591
  178   G  HA3      3.9588
  179   N  N      119.0393
  179   N  CA      52.5191
  179   N  CB      39.2328
  179   N  C      175.2298
  179   N  H        8.1587
  179   N  HA       4.5801
  179   N  HB2      2.8803
  179   N  HB3      2.7315
  179   N  HD21     7.1239
  179   N  HD22     6.6813
  180   L  N      129.1721
  180   L  CA      57.2827
  180   L  CB      41.6144
  180   L  C      179.4449
  180   L  H        8.4596
  180   L  HA       3.9647
  180   L  HB2      1.7233
  180   L  HB3      1.7012
  180   L  HD11     0.9183
  180   L  HD12     0.9181
  180   L  HD13     0.9184
  180   L  HD21     0.2286
  180   L  HD22     1.8085
  180   L  HD23     0.8672
  180   L  HG       1.7020
  181   A  N      122.0420
  181   A  CA      54.6621
  181   A  CB      18.5953
  181   A  C      179.7090
  181   A  H        7.8465
  181   A  HA       3.9749
  181   A  HB1      1.3853
  181   A  HB2      1.3498
  181   A  HB3      1.3304
  182   C  N      116.6791
  182   C  CA      60.1073
  182   C  CB      43.4588
  182   C  C      175.8871
  182   C  H        7.9861
  182   C  HA       4.1470
  182   C  HB2      2.9961
  182   C  HB3      3.2248
  183   L  N      119.4380
  183   L  CA      57.7229
  183   L  CB      41.1953
  183   L  C      179.3353
  183   L  H        7.9881
  183   L  HA       3.9213
  183   L  HB2      1.8667
  183   L  HB3      1.7133
  183   L  HD11     0.9171
  183   L  HD12     0.9908
  183   L  HD13     0.9013
  183   L  HD21     0.4346
  183   L  HD22     2.3547
  183   L  HD23     0.9101
  183   L  HG       1.6918
  184   S  N      114.3641
  184   S  CA      61.5197
  184   S  CB      62.6505
  184   S  C      176.6456
  184   S  H        8.0123
  184   S  HA       4.2536
  184   S  HB2      4.0586
  184   S  HB3      4.2797
  185   L  N      121.9255
  185   L  CA      56.5343
  185   L  CB      41.5781
  185   L  C      174.9065
  185   L  H        7.5468
  185   L  HA       4.1438
  185   L  HB2      1.5743
  185   L  HB3      1.7423
  185   L  HD11     0.9087
  185   L  HD12     0.9097
  185   L  HD13     0.9119
  185   L  HD21     0.3486
  185   L  HD22     1.5025
  185   L  HD23     0.8614
  185   L  HG       1.7924
  186   C  N      111.7517
  186   C  CA      54.8638
  186   C  CB      42.0864
  186   C  C      171.6661
  186   C  H        7.2764
  186   C  HA       5.1504
  186   C  HB2      3.3341
  186   C  HB3      3.2617
  187   H  N      116.6135
  187   H  CA      56.2476
  187   H  CB      29.2663
  187   H  C      175.1253
  187   H  H        7.5957
  187   H  HA       4.4410
  187   H  HB2      3.1324
  187   H  HB3      3.2285
  187   H  HD2      5.8606
  187   H  HE1      6.9931
  188   I  N      123.4604
  188   I  CA      60.2878
  188   I  CB      41.8167
  188   I  C      175.7561
  188   I  H        7.3063
  188   I  HA       4.4840
  188   I  HB       1.8931
  188   I  HD11     0.8689
  188   I  HD12     0.8672
  188   I  HD13     0.8661
  188   I  HG12     1.2828
  188   I  HG13     0.7957
  188   I  HG21     0.9369
  188   I  HG22     0.9250
  188   I  HG23     0.9217
 *189   E  N      118.8058
 *189   E  CA      56.1939
 *189   E  CB      30.2523
 *189   E  C      176.7713
 *189   E  H        8.6420
 *189   E  HA       4.1708
 *189   E  HB2      1.7788
 *189   E  HB3      1.7525
 *189   E  HG2      2.1312
 *189   E  HG3      2.3767
